.github/workflows/prerelease.yml

```diff
@@ -9,25 +9,20 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  GO_VERSION: '1.9.7'
+  GO_VERSION: '1.16'
   INTEGRATION: "apache"
   ORIGINAL_REPO_NAME: ${{ github.event.repository.full_name }}
   REPO_FULL_NAME: ${{ github.event.repository.full_name }}
   TAG: ${{ github.event.release.tag_name }}
 
 jobs:
-  validate:
-    name: Validate code via linters
-    runs-on: ubuntu-20.04
+  golangci-lint:
```

Comment:
don't forget to update the conditions on the following jobs 
https://github.com/newrelic/nri-apache/pull/59/files#diff-884bb2e52646d4b226c75e5dd7c196f9d654accb529fef4b6eeb401daa83144eR102

Reply:
line 107 of this file